google/cloud/policytroubleshooter/doc/main.dox

```diff
@@ -81,7 +83,7 @@ The library automatically retries requests that fail with transient errors, and
 uses [exponential backoff] to backoff between retries. Application developers
 can override the default policies.
 
-[cloud-service-docs]: https://cloud.google.com/policytroubleshooter
+[cloud-service-docs]: https://cloud.google.com/iam/docs/troubleshooting-access
```

Comment:
Since it is under the `iam` product umbrella we could have added this to the existing IAM library.  We can still change this before declaring GA.  I will add a discussion item for the team meeting, we can merge as-is.